app/cmd/snapshot.go

```diff
@@ -415,3 +459,56 @@ func cloneSnapshotStatus(c *cli.Context) error {
 	fmt.Println(string(output))
 	return nil
 }
+
+func hashSnapshot(c *cli.Context) error {
+	url := c.GlobalString("url")
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	task, err := sync.NewTask(ctx, url)
+	if err != nil {
+		return err
+	}
+
+	names := c.StringSlice("snapshot-name")
+	if names == nil {
+		return errors.Wrap(err, "snapshot-name is required")
+	}
+
+	rehash := c.Bool("rehash")
+
+	concurrentLimit := c.Int("concurrent-limit")
+	if concurrentLimit < 0 {
```

Comment:
Will it make sense to auto-correct to `0` if `< 0`?

Reply:
Yeah, the auto-correctness is okay.

Reply:
Auto-correct ot 0 if < 0. Thank you.